resources/js/admin/users/components/SecurityLogsModal.vue

```diff
@@ -223,8 +229,8 @@ export default {
       return auxArraySorted;
     },
     /**
-     * Verify if value is a string o null
```

Comment:
Please not remove the CS

Reply:
OK, that Code was 100% my code that's why I applied CS. I'll be careful with foreign code.